src/Event.cpp

```diff
@@ -2,8 +2,8 @@
 
 #include "Event.h"
 
-Event::Event(std::string&& name)
-    : eventName(name) {
+Event::Event(EventType eType)
+    : eventType(eType) {
```

Comment:
So I actually named everything as snake case in my original PR before I could comment on this, how do we want to move forward then. camelCase or snake_case in the future for function and variable names?